### PR TITLE
Initial GithubActions support

### PR DESCRIPTION
**Pull request by artemdinaburg**
Remove Travis, Add GithubActions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,11 +82,16 @@ To try out Rellic you can do the following, given a LLVM bitcode file of your ch
...
 
-The Docker image should provide an environment which can set-up, build, and run rellic.
+The Docker image should provide an environment which can set-up, build, and run rellic. The Docker images are parameterized by Ubuntu verison, LLVM version, and architecture.
 
-To build the docker image:
+To build the docker image using LLVM 8.0 for Ubuntu 18.04 on amd64 you can run the following command:
 ```sh
-docker build . -t rellic:llvm800-ubuntu18.04-amd64 -f Dockerfile --build-arg UBUNTU_VERSION=18.04 --build-arg ARCH=amd64 --build-arg LLVM_VERSION=800
+ARCH=amd64; UBUNTU=18.04; LLVM=800; docker build . \
+  -t rellic:llvm${LLVM}-ubuntu${UBUNTU}-${ARCH}\
```
Comment (ekilmer) [inline README.md:90]: Not sure if this actually matters but for consistency
```suggestion
  -t rellic:llvm${LLVM}-ubuntu${UBUNTU}-${ARCH} \
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        CTEST_OUTPUT_ON_FAILURE=1 cmake --build . --target install
+        cd ..
+    - name: Test Rellic Build
+      run: |
+        scripts/roundtrip.py /opt/trailofbits/rellic/bin/rellic-decomp-8.0 $(realpath tests/tools/decomp) $(which clang-8)
+  Docker_Linux:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        llvm: ["800"]
```
Comment (ekilmer) [inline .github/workflows/ci.yml:31]: Do we want to test more than just LLVM 800?

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Native LLVM 8.0 on Ubuntu 18.04 Build
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y ninja-build unzip libomp-dev libomp5 libtinfo-dev lsb-release
+        scripts/build.sh --llvm-version llvm800 --prefix /opt/trailofbits/rellic --use-host-compiler --extra-cmake-args "-DCMAKE_BUILD_TYPE=Release"
+        cd rellic-build
+        CTEST_OUTPUT_ON_FAILURE=1 cmake --build . --verbose --target test
+        CTEST_OUTPUT_ON_FAILURE=1 cmake --build . --target install
```
Comment (ekilmer) [inline .github/workflows/ci.yml:22]: Probably don't need this
```suggestion
        cmake --build . --target install
```

File: .github/workflows/ci.yml (hunk 2 above)
Comment (artemdinaburg) [reply .github/workflows/ci.yml:31]: We do when they are supported :)

Right now Rellic only builds against LLVM8 due to LLVM API differences

**Pull request merged**